omnibus/config/projects/push-jobs-client.rb

```diff
@@ -24,7 +24,7 @@
 conflict "opscode-push-jobs-client"
 
 build_iteration 1
-build_version "2.0.2"
+build_version "2.1.0"
```

Comment:
I changed the ruby being shipped from 2.1.5 from rubyinstaller to 2.1.8 that we compile.  Also we don't compile devkit.  Now I'm pretty sure devkit was not part of the push jobs API but I figured it was safe to just bump this sucker and not worry about it...